NICE.Identity.Management/ClientApp/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+	parser: "babel-eslint",
+	extends: ["@nice-digital/eslint-config/es6", "plugin:react/recommended"],
```

Comment:
I think we should probably use plugin:@typescript-eslint/recommended or similar for linting our typescript